Microsoft.Azure.CosmosRepository/src/DefaultRepository.cs

```diff
@@ -223,6 +223,20 @@ await container.UpsertItemAsync<TItem>(value, new PartitionKey(value.PartitionKe
             return results;
         }
 
+        public async ValueTask<int> CountAsync(Expression<Func<TItem, bool>> predicate,

```

Comment:
It would be good to see this adding to one of the samples, the `WebTier` may be a good start?